inst/bin/codemeta-description-updated

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# adapted from https://github.com/lorenzwalthert/precommit/blob/f4413cfe6282c84f7176160d06e1560860c8bd3d/inst/bin/readme-rmd-rendered
+
+if [[ DESCRIPTION -nt codemeta.json ]]
+then
+    echo >&2 "codemeta.json is out of date; please re-run codemetar::write_codemeta()."
+    exit 1
```

Comment:
why not running it yourself here? I think that's what we do with all other hooks. So after a failed commit, you can inspect the new diff and try to commit right away.

Reply:
Because it can take a while if the internet connection is poor (ask me how i know 😭)

Reply:
So what's the workflow then in that case for the user if not manually re-run? Wait with committing until better internet connection? Skip the hook? 

We could also think about using hashing of relevant DESCRIPTION sections in text in order to know if a hook must run or not, to prevent false positive runs (i.e. you change a field in DESCRIPTION that does not affect codemeta.json.

Reply:
The user would skip it I suppose. 

Most fields in DESCRIPTION will affect codemeta.json.